src/RenameMe/SupportFileDownloads.php

```diff
@@ -20,7 +20,13 @@ function __construct()
 
             $response = $returned;
 
-            $name = Str::after($response->headers->get('Content-Disposition'), 'filename=');
+            preg_match(
```

Comment:
Please extract this to a separate function, and maybe add a comment to accompany the regex expression - it's not obvious at first glance

Reply:
Here is my first attempt.  I also simplified the regexes for more readability since we are in a new method now.

Reply:
This is much better, thanks!